### ISSUE TITLE
Submit a Project - Remove extraneous empty IMG tag

### ISSUE THREAD

**Issue by terrimorton** [dotnet-foundation/website#304]
Page: https://dotnetfoundation.org/projects/submit

I found this issue when running Accessibility Insights for Web on this page. The extraneous IMG tag should be removed from the end of the "Why you'd select it" column heading.

`<th>Why you'd select it <img width="900/"></th>`

![image](https://user-images.githubusercontent.com/4643903/90282247-00bcd200-de3c-11ea-846e-63277159158a.png)

**Comment by Arhell** [comment]
@terrimorton  done

**Comment by terrimorton** [comment]
@clairernovotny @Arhell It's been 5 hours and I still don't see the change at https://dotnetfoundation.org/projects/submit.

**Comment by Arhell** [comment]
Strange, but I no longer have this hidden picture
try pressing ctrl + shift + r
maybe you still have browser cache

**Comment by terrimorton** [comment]
@Arhell @clairernovotny I had cleared browser cache a bunch of times - ctrl_shift+r as well as ctrl+f5.   I just now revisited the page and the image tag is gone. Not sure what was going on but it's all good now. Thanks for the PR!
